### PR TITLE
Fixes #20307 - Add unicode char check to certs check

### PR DESCRIPTION
**Pull request by chris1984**
Fixed a few grammer mistakes as well on some of the other functions.

### CODE REVIEW

**Comment by chris1984** [comment]
Example of output:

Checking for non unicode characters[FAIL]
There are non unicode characters present:

/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem:1426:# Certinomis - Autorité Racine /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem:3058:# NetLock Arany (Class Gold) Főtanúsítvány /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem:4223:# TÜRKTRUST Elektronik Sertifika Hizmet Sağlayıcısı /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem:4395:# TÜBİTAK UEKAE Kök Sertifika Hizmet Sağlayıcısı - Sürüm 3 /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem:4427:# TÜRKTRUST Elektronik Sertifika Hizmet Sağlayıcısı H5 /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem:4454:# TÜRKTRUST Elektronik Sertifika Hizmet Sağlayıcısı H6

**Comment by chris1984** [comment]
@ekohl can I get a review please :beers: build failure is unrelated.

**Comment by ekohl** [comment]
https://github.com/Katello/puppet-katello/pull/198 and https://github.com/Katello/puppet-foreman_proxy_content/pull/129 should fix the build issue.

File: bin/katello-certs-check [hunk 1]
```diff
@@ -112,11 +112,24 @@ function check-ca-bundle () {
     fi
 }
 
+function check-non-unicode () {
```
Comment (ekohl) [inline bin/katello-certs-check]: Do you mean non-ascii instead of non-unicode?

**Comment by chris1984** [comment]
[test]

**Comment by chris1984** [comment]
@ekohl can I get another review please, also if it looks good can you kick off the travis job again I dont have the ability to.

**Comment by ekohl** [comment]
I'm not sure why it's still not able to do a build, but that's not related to this PR.

**Comment by chris1984** [comment]
@ekohl cool thanks for the review :beer: since its not related did you want want for it to get fixed before merge?

**Comment by chris1984** [comment]
@ekohl since the tests are not related did you want to merge or hold off and wait?

**Comment by ekohl** [comment]
I was hoping for a second ACK but I don't mind merging.

**Pull request merged**